ros_buildfarm/templates/devel/devel_create_tasks.Dockerfile.em

```diff
@@ -77,12 +77,14 @@ cmd = \
     ' --build-tool ' + build_tool + \
     ' --ros-version ' + str(ros_version) + \
     ' --env-vars ' + ' ' .join(env_vars)
+if run_abichecker:
+    cmd += " --run-abichecker"
 cmds += [
     cmd +
     ' --dockerfile-dir /tmp/docker_build_and_install',
     cmd +
     ' --dockerfile-dir /tmp/docker_build_and_test' +
-    ' --testing',
+    ' --testing'
```

Comment:
If there's no need to remove the trailing comma may as well just keep it.

Reply:
ouch, reverted 5e6f456e4c5caa0581de56d51cba3283c8cdc696